src/website/views/components/activity.pug

```diff
@@ -0,0 +1,1105 @@
+            </div>
+          </div>
+          <div class="m-activity__item">
+            <div class="m-activity__item-avatar">
+              <div class="a-avatar">
+                <img src="../../assets/images/avatar.jpg" alt="avatar">
+              </div>
+            </div>
+            <div class="m-activity__end">
+              <div class="m-activity__end-header"><strong>trent.anderson</strong> deployed<strong> Active Directory</strong>
+              </div>
+              <div class="m-activity__end-footer">04/22/2019 · 2:37 PM</div>
+            </div>
+          </div>
+          <div class="m-activity__item">
+            <div class="m-activity__item-avatar">
+              <div class="a-avatar">
+                <img src="../../assets/images/avatar.jpg" alt="avatar">
+              </div>
+            </div>
+            <div class="m-activity__end">
+              <div class="m-activity__end-header"><strong>trent.anderson</strong> deployed<strong> Active Directory</strong>
+              </div>
+              <div class="m-activity__end-footer">04/16/2019 · 5:00 PM</div>
+            </div>
+          </div>
+          <div class="m-activity__item">
+            <div class="m-activity__item-avatar">
+              <div class="a-avatar">
+                <img src="../../assets/images/avatar.jpg" alt="avatar">
+              </div>
+            </div>
+            <div class="m-activity__end">
+              <div class="m-activity__end-header"><strong>trent.anderson</strong> deployed<strong> Active Directory</strong>
+              </div>
+              <div class="m-activity__end-footer">04/16/2019 · 5:00 PM</div>
+            </div>
+          </div>
+          <div class="m-activity__item">
+            <div class="m-activity__item-avatar">
+              <div class="a-avatar">
+                <img src="../../assets/images/avatar.jpg" alt="avatar">
+              </div>
+            </div>
+            <div class="m-activity__end">
+              <div class="m-activity__end-header"><strong>trent.anderson</strong> deployed<strong> Active Directory</strong>
+              </div>
+              <div class="m-activity__end-footer">04/16/2019 · 11:20 AM</div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+h4 Feed with stories
+.example.-mb--3
+  .-p--5
+    .m-activity.-feed.-stories
+      .m-activity__day
+        .m-activity__content
+          .m-activity__item
+            .m-activity__start
+              div
+                .a-avatar
+                  img(src='../../assets/images/avatar.jpg', alt="avatar")
+            .m-activity__end
+              .m-activity__end-header
+                strong trent.anderson
+                |  deployed
+                strong  Active Directory
+              .m-activity__end-content
+                ul.-stories
+                  li
+                    div
+                      .a-icon.-text--primary
```

Comment:
There's 16 occurrences of `-text--primary` used on checkmark icons in the feed examples. Can you change them all to `-text--success` to match the design?